Dockerfile

```diff
@@ -20,28 +20,41 @@ ENTRYPOINT [ "/sbin/tini", "--"]
 # Set Working Directory Context
 WORKDIR "/telescope"
 
-# Copy shared context
+# Copy package.jsons for each service
 COPY package.json .
+COPY ./tools/autodeployment/package.json ./tools/autodeployment/package.json
+COPY ./src/frontend/package.json ./src/frontend/package.json
 
 # -------------------------------------
 # Context: Dependencies
 FROM build AS dependencies
 
-# Get Entire Codebase
+# Install Production Modules! 
+# Disable postinstall hook in this case since we are being explict with installs
+RUN npm install --only=production --no-package-lock --ignore-scripts
+
+# Install Frontend Modules!
+RUN cd ./src/frontend && npm install --no-package-lock
```

Comment:
Why won't this get triggered by the `postinstall` script after 34?

Reply:
`--ignore-scripts` causes all hooks to be skipped. This enables me to be very explicit with the installation calls.

Reply:
Doing a clean build now to confirm this, want to go over logs and ensure I am correct on that logic.